django_lightweight_queue/logger.py

```diff
@@ -0,0 +1,10 @@
+from typing import TypeVar, Callable, Iterable, NamedTuple
+
+T = TypeVar('T')
+
+Logger = NamedTuple('Logger', [
```

Comment:
Minor: I think using this name could be confusing and imply that this is a logger as in a `logging` logger. `ProgressLogger` would be more explicit but sounds weird.

(not just the type, this carries across the PR but had to pick a place to comment)

Reply:
db1258e